Explain how to contribute to the orientation guide

robmoss opened the pull request:
This adds an admonition to the start of the Orientation Guide, which explains how to suggest topics (create issues) and add content (via pull requests).

robmoss committed:
Explain how to contribute to the orientation guide

robmoss commented:
Follow [this link](https://git-is-my-lab-book.net/pr-preview/pr-68/orientation/) to see the admonition.

robmoss commented:
@EamonConway apologies for the short notice, if possible it would be great to merge this before our CoP meeting (2pm today).

robmoss committed:
Refer to our "how to contribute" instructions

robmoss committed:
Add a link to existing orientation guide issues

robmoss committed:
Detailed instructions for different contributions

The "How to contribute" box now lists different steps for suggesting new
topics, providing feedback about existing content, and contributing new
content.

robmoss committed:
Add instructions for suggesting a useful resource

robmoss commented:
@EamonConway hang on, I'll make the "submit an issue" link text more descriptive ...

robmoss committed:
Use descriptive text for each new issue link

robmoss commented:
@EamonConway all done, ready for you to review :)

EamonConway commented on docs/orientation/README.md:
```diff
@@ -3,6 +3,28 @@
+
+    * Use the search box (top of the page) and check if similar content already exists;
+
+    * If there is no similar content, please fork [the repository](https://github.com/robmoss/git-is-my-lab-book), add your contributions, and create a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
```
Do we want to provide a similar link (https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) for the forks as we do pull request?

robmoss replied:
Great suggestion, thanks very much @EamonConway! I've added the link, and will merge the PR. Thanks again :)

EamonConway reviewed (commented):
Hi Rob,
I have gone through the review and am happy with it, flows quite nicely and all hyperlinks are working correctly. 
I have provided one comment about forks, and am happy with either decision you make.

robmoss committed:
Link to GitHub documentation about creating forks

We already linked to GitHub documentation about creating pull requests,
but the first step is to fork the main repository. So we now have a link
to the relevant documentation here, as suggested by Eamon.
